CHANGELOG.md

```diff
@@ -25,6 +25,37 @@ playback to start before the entire session is downloaded and unpacked.
 Additionally, `tsh play` now supports a `--speed` flag for adjusting the
 playback speed.
 
+#### Teleport Operator can now be deployed against a remote Teleport cluster
+
+Prior to Teleport 15, the Teleport Kubernetes Operator had to run as a sidecar
+of the Teleport auth. It was not possible to use the operator in Teleport Cloud
+or against a Teleport cluster not deployed with the `teleport-cluster` Helm chart.
+
+In Teleport 15, the Teleport Operator can reconcile resource in any Teleport
+cluster. Teleport Cloud users can now use the operator to manage their resources.
+
+When deployed with the `teleport-cluster` chart, the operator now runs in a
+separate pod. This ensures that Teleport's availability won't be impacted if
+the operator becomes unready.
+
+See [the Standalone Operator guide](docs/pages/management/dynamic-resources/teleport-operator-standalone.mdx)
+for installation instructions.
+
+#### Teleport Operator now supports roles v6 and v7
+
+Starting with Teleport 15, newly supported kinds will contain the resource version.
+For example: `TeleportRoleV6` and `TeleportRoleV7` kinds will allow users to
+create Teleport Roles v6 and v7.
+
+Existing kinds will remain unchanged in Teleport 15, but will be renamed in
+Teleport 16 for consistency.
+
+To migrate an existing `TeleportRole` to a `TeleportRoleV7`, you must:
+- upgrade Teleport and the operator to v15
+- annotate the Custom Resource with `teleport.dev/keep: "true"`
+- delete the Custom Resource (it won't delete the resource thanks to the annotation)
+- create a new `TeleportRoleV7` with the same name
```

Comment:
We need to take some care here because some default values might changed across versions and we need to keep the compatibility

Reply:
The flow would be:
- read the documentation to know the difference between role versions
- ...
- write documentation to know the difference between role versions 🙃

Reply:
Joke aside, is the v5 to v7 migration breaking things?

I know that v5 -> v6 removes the kubernetes_access wildcard and v6->v7 brings it back and adds support to every resource.

From what I understand, you can migrate from v5 to v7 without breaking too much stuff; what edge case am I missing?

Reply:
If you have a kubernetes v5/v6 role with:

```
kubernetes_resources:
- name: "*"
  namespace: 'some_namespace'
  kind: pod
``` 
It blocks access to pods on all namespaces besides `some_namespace`. For all other resources, it **ALLOWS** unrestricted access.

If you copy and paste that into a v7 role, 
It blocks access to pods on all namespaces besides `some_namespace`. For all other supported resources, it **DENIES** access.

To convert it to v7 correctly,

```
kubernetes_resources:
- name: "*"
  namespace: 'some_namespace'
  kind: pod
- name: "*"
  namespace: '*'
  kind: secret
- name: "*"
  namespace: '*'
  kind: clusterrole
...
```

Reply:
Got it. I made a doc page explaining these differences. I'll merge this PR and poke you in the other one.